plugins/github/personal_access_token_test.go

```diff
@@ -85,6 +85,31 @@ func TestPersonalAccessTokenImporter(t *testing.T) {
 				},
 			},
 		},
+		"GitHub config file with ghp_ token prefix": {
```

Comment:
Given that this is the happy path:
```suggestion
		"GitHub config file": {
```